dasharo-security/tpm-support.robot

```diff
@@ -146,3 +147,50 @@ TPM003.003 Check TPM Physical Presence Interface (Windows 11)
 #    Skip If    not ${tpm_support}    TPM003.004 not supported
 #    Skip If    not ${tests_in_ubuntu_support}    TPM003.004 not supported
 # TODO: https://docs.dasharo.com/unified-test-documentation/dasharo-security/200-tpm-support/#tpm003004-change-active-pcr-banks-with-tpm-ppi-firmware
+
+TPM004.001 Check TPM Clear procedure
+    [Documentation]    This test aims to verify whether the TPM Clear procedure
+    ...    works properly
+    Skip If    not ${TPM_SUPPORT}    TPM004.001 not supported
+    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    TPM004.001 not supported
+
+    Power On
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    Detect Or Install Package    tpm2-tools
+
```

Comment:
@PLangowski I think we should start this test by clearing TPM as we do not know the state of the ownership at this time.